### ISSUE TITLE
How to use docker image?

### ISSUE THREAD

**Issue by drnic** [pivotal/LicenseFinder#332]
At a guess, I assumed that the docker image would have `license_finder` CLI in its path:

```
$ docker run -ti licensefinder/license_finder license_finder
docker: Error response from daemon: oci runtime error: container_linux.go:262: starting container process caused "exec: \"license_finder\": executable file not found in $PATH".
```

What is the way that works for using the docker image?

**Comment by ryanmattcollins** [comment]
We actually normally mount our local workspace to the docker container and run `/bin/bash --login`. Then we `cd` into the directory in which we want licenses found, and run the `license_finder` cli.

```
docker run -v ~/workspace/:/workspace/ -it licensefinder/license_finder /bin/bash --login
cd /workspace/licensefinderable-project
license_finder
```

Out of curiosity, what would you expect the behaviour to be if you ran the CLI directly like you did?

**Comment by kdykeman** [comment]
@drnic We just released version 3.0.4. With this release, we also added a wrapper script (dlf) that helps run license_finder in docker. It mounts your current working directory as /scan in the docker image.  There's a bit more info in [README.md](https://github.com/pivotal/LicenseFinder/blob/master/README.md), but you should essentially be able to run `dlf "bundle install && license_finder report"`

**Comment by matthiasbalke** [comment]
> Out of curiosity, what would you expect the behaviour to be if you ran the CLI directly like you did?

I would expect that, the license_finder binary is executed in the `/scan` directory. So I could run the license analysis just like:

```shell
docker run -v /path/to/my/project:/scan licensefinder/license_finder
```
